installer/packer/scripts/systemd/admiral/admiral.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Admiral is a highly scalable and very lightweight Container Management platform for deploying and managing container based applications.
+Documentation=https://github.com/vmware/admiral
+After=docker.service systemd-networkd.service systemd-resolved.service
+Requires=docker.service
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=5
+ExecStart=/usr/local/bin/docker run -d -p 8282:8282 --name vic-admiral docker.io/vmware/admiral:dev
```

Comment:
It is better to run Admiral with the log parameters suggested in the config doc:

--log-driver=json-file --log-opt max-size=1g --log-opt max-file=10

may be even less than 1g for max-size